Week1/homework/js-exercises/ex6.js

```diff
@@ -0,0 +1,17 @@
+//Exercise 6: Log an array of animals
+//Declare variable and assign to it an empty array
+const empty = [, , ,];
```

Comment:
Is this really an empty array? Look closely at what is logged on line 7.

Reply:
when i do [, , ,] its open the square brackets and give me empty word but when i open it with nothing []; its give me only [] in terminal didn't says its empty, changes done to [];